app/ide-desktop/eslint.config.js

```diff
@@ -226,6 +226,18 @@ export default [
             ...tsEslint.configs['recommended-requiring-type-checking']?.rules,
             ...tsEslint.configs.strict?.rules,
             eqeqeq: ['error', 'always', { null: 'never' }],
+            'require-jsdoc': [
+                'error',
+                {
+                    require: {
+                        FunctionDeclaration: true,
+                        MethodDefinition: true,
+                        ClassDeclaration: true,
+                        ArrowFunctionExpression: true,
```

Comment:
I think that requiring docs on arrow fn exprs is an overkill. Vast majority of them are small helper functions that do not require better docs than just a good name. In case we will find a code during review with too long such an arrow expression, we can always refactor it to a normal fn :)